### DISCUSSION TITLE
Terraform Lockfile maintenance

### DISCUSSION THREAD

**Discussion by renan-alm** [renovatebot/renovate#11521] [Request Help] [answered]
Hi,

I'm currently using a self-hosted instance of Renovate to monitor Terraform and I would like to ask if there is any way to make Renovate provide locks for several platforms while updating dependencies. 

Currently, these changes are being made manually on top of PRs and I couldn't find any feature on that on the documentation.

`terraform providers lock -platform=windows_amd64 -platform=darwin_amd64 -platform=linux_amd6`

Is there such a feature?

Kind regards,
Renan A.

**Comment by viceice** [comment, marked as answer]
yes, since [v25.63.0](https://github.com/renovatebot/renovate/releases/tag/25.63.0), but you should use [v26.14.1](https://github.com/renovatebot/renovate/releases/tag/26.14.1)  or newer because of #11502
